eo-maven-plugin/src/test/java/org/eolang/maven/OyCachingTest.java

```diff
@@ -45,15 +45,15 @@ void putsObjectToLocalCache(@TempDir final Path path) throws Exception {
         MatcherAssert.assertThat(
             new TextOf(
                 new OyCaching(
-                    "master",
+                    "branch",
```

Comment:
@yasamprom Is this change really necessary?

Reply:
> @yasamprom Is this change really necessary?

No, it's not. I just experimented with names and commited it by chance. However, all tests passed. Should I change it back?